runtime/onert/core/include/backend/ITensorRegistry.h

```diff
@@ -33,10 +35,17 @@ struct ITensorRegistry
   virtual ~ITensorRegistry() = default;
 
   /**
-   * @brief Returns pointer of ITensor
+   * @brief Returns pointer of ITensor among managed and external tensors
```

Comment:
does this `external tensors` mean `External Tensor` in #3119?

Reply:
No. More precisely, it is "portable tensors that are from other backends".

Reply:
When I first read this comment, I also thought that `ExternalTensor` in #3119. :-P 
Maybe we need different term since #3119 obtained copyright(?) of external tensor. :-)

Reply:
For different term, I have another question. What does `managed` mean?

Reply:
- Managed : All tensors that the backend directly created
- External : Tensors(portable) that are created by other backends
- Managed ∪ External : All tensors that are accessible from the backend

Sometimes we need to check if a tensor is managed or external.

Reply:
I have no idea. "native" and "immigrant"?